cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1663,7 +1663,7 @@ endif()
 #---Check for VecGeom--------------------------------------------------------------------
 if (vecgeom)
   message(STATUS "Looking for VecGeom")
-  find_package(VecGeom ${VecGeom_FIND_VERSION} CONFIG QUIET)
+  find_package(VecGeom 1.2 CONFIG)
```

Comment:
In principle the move happened in version 1.1.5. Do we want to support these older versions?

Reply:
No, let's leave it from the major version